### PR TITLE
Issue 1065 - Import missing Pages

### LINKED ISSUE #1065: Import missing Pages
- [ ] /app-note/br/phenotypic-profiling-of-cftr-modulators-in-patient-derived-respiratory-epithelia
- [ ] /explore-pathways-pro
- [ ] /newsroom/in-the-news/brockway-alum-previews-road-to-ph-d-to-future-graduates
- [ ] /newsroom/in-the-news/spring-awakening-salzburg
- [ ] /newsroom/in-the-news/where-are-hottest-drug-discovery-hubs-in-the-us

### PR DESCRIPTION
**Pull request by alexcarol**
Fix #1065 

Test URLs:
Explore pathways pro (embed - https://live-md30.pantheonsite.io/explore-pathways-pro):
Before: https://main--moleculardevices--hlxsites.hlx.page/drafts/acarol/explore-pathways-pro-copy
After: https://issue-1065--moleculardevices--hlxsites.hlx.page/drafts/acarol/explore-pathways-pro-copy

Phenotypic profiling page:
Before: https://main--moleculardevices--hlxsites.hlx.page/app-note/br/phenotypic-profiling-of-cftr-modulators-in-patient-derived-respiratory-epithelia
After: https://issue-1065--moleculardevices--hlxsites.hlx.page/app-note/br/phenotypic-profiling-of-cftr-modulators-in-patient-derived-respiratory-epithelia

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -819,6 +819,19 @@ main .section.tabs[aria-labelledby="order"] .default-content-wrapper img {
   font-size: 36px;
 }
 
+.section.small-text {
```
Comment (andreituicu) [inline styles/styles.css:822]: Would citation work instead of small text? We already have stylings for citions that look very similar.

Comment (andreituicu) [reply styles/styles.css:822]: Example at the bottom of https://www.moleculardevices.com/lab-notes/clone-screening/how-to-develop-monoclonal-cell-lines-for-stem-cell-engineering (even if it is a list)

Comment (alexcarol) [reply styles/styles.css:822]: The problem with "text-caption" (assuming that's the one you mean) is that it's meant for lists, and that it also makes the text italic. I can turn that text into a list in that case, but I also added small-text to use it in a list that is *not* in italics (in that case text-caption would not work).

**Review by andreituicu — APPROVED** [review]

File: styles/styles.css (hunk 1 above)
Comment (andreituicu) [reply styles/styles.css:822]: I understand. we can keep it like this.

**Pull request merged**